src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    {age > 0 && (
+      <p className="Person__age">{`I am ${age}`}</p>
+    )}
+
+    <p className="Person__partner">
+      {isMarried
+        ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
using ternary operator inside another one is not good idea

for better readability you can create separate variable in component like `partnerStatus` and use it in jsx
```suggestion
  const partnerStatus = sex === 'm'
     ? 'wife'
     : 'husband';
     ...
     `${partnerName} is my ${partnerStatus}`
```